spec/lib/miq_environment_spec.rb

```diff
@@ -10,8 +10,10 @@
     end
 
     context "Host Info" do
-      example "fully_qualified_domain_name" do
-        expect(described_class.fully_qualified_domain_name).to eq(`hostname -f`.chomp)
+      unless RUBY_PLATFORM.include?("darwin")
+        example "fully_qualified_domain_name" do
+          expect(described_class.fully_qualified_domain_name).to eq(`hostname -f`.chomp)
+        end
```

Comment:
As per my suggestion above (which is marked outdated for some reason

```suggestion
      example "fully_qualified_domain_name" do
        skip if RUBY_PLATFORM.include?("darwin")
        expect(described_class.fully_qualified_domain_name).to eq(`hostname -f`.chomp)
      end
```